_posts/2016-10-16-getting-started-in-scala-on-android.md

```diff
@@ -0,0 +1,217 @@
+```
+
+##Integration with Android Studio
+From this point, you could develop your Android apps in Scala using a text editor and sbt. But it would be good to be able to use the official IDE, which offers a lot of useful tools. It's also possible to use IntelliJ, but I won't go into this detail in this post.
+
+Before we start, we'll need to install both the [Scala](https://plugins.jetbrains.com/plugin/1347) and [SBT plugins](https://plugins.jetbrains.com/plugin/5007) for Android Studio.
+
+To import our project, open Android Studio and select `Import project (Eclipse, ADT, Gradle, etc.)`.
+
+Select `Import project from external module`, and `SBT`:
+
+<img src="/assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/scala-on-android_import-sbt.png" alt="Import project" class="img-responsive"/>
+
+Select your Android SDK on the Project SDK option, and check `Sources`, `Javadocs` and `Sources for SBT and plugins`:
+
+<img src="/assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/scala-on-android_import-sdk.png" alt="Import project" class="img-responsive"/>
+
+On `Project Settings`/`Modules` select the project module and click on `+`, and Add `Android` under the `Framework` section. On the Structure tab, replace all `/.idea/modules` paths with `/src/main`:
+
+<img src="/assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/scala-on-android_proj-structure-module.png" alt="Project structure" class="img-responsive"/>
+
+Now that the project is set up, we can try to run it. Create a new Run/Debug Configuration. Select `Android App` and give it a name (e.g. `app`).
+
+On the `General` tab, select the Module. We also need to configure how to run the app with SBT. On the `Before launch` section, remove the default `Gradle-aware Make` task clicking on `-`:
+
+<img src="/assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/scala-on-android_run-debug-config.png" alt="Run/Debug configuration" class="img-responsive"/>
+
+Finally, click on `+`, create a new `SBT` task, and add `android:run`. Leave the `Run in current module` option checked:
+
+<img src="/assets/img/custom/blog/2016-10-16_getting-started-in-scala-on-android/scala-on-android_run-before-launch-sbt-android-run.png" alt="Before launch SBT android run" class="img-responsive"/>
+
+From now, you can run the application in the same way as any app written in Java or Kotlin.
+
+##Conclusion
+Setting up Scala and SBT on Android is not as straightforward as setting up Java or Kotlin and Gradle, but can it be done. This opens the door to a new world of possibilities offered by Scala and its powerful features and build system. Kotlin still looks like a better fit for Android (easier setup, better IDE support, and smaller runtime, among others), but having the possibility of using Scala and SBT can bring developers who build their backends in Scala, or want to use high level Functional Programming features to build complex apps.
```

Comment:
The last sentence is a little long. How about
`Kotlin still looks like a better fit for Android: Among other benefits, it has an easier setup, better IDE support and smaller runtime. However the possibility of using Scala and SBT can allow developers who build their backends in Scala or want to use high level Functional Programming features to build complex apps.`